[TASK] improve delete message for tags/fieldcollections/objectbricks/…

dpfahlbusch opened the pull request:
…users and add confirmdialog to thumbnails/customreports

## Changes in this pull request  
Partly Resolves #6259

## Additional info

Commit:
[TASK] improve delete message for tags/fieldcollections/objectbricks/users and add confirmdialog to thumbnails/customreports

Partly resolves: #6259

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/pimcore/pimcore?pullRequest=12102) <br/>All committers have signed the CLA.

Commit:
[TASK] add confirm dialogs to classificationstore delete operations (key/relation/collection/group)

Relates: #6259

Commit:
[TASK] add confirm dialogs videothumbnail delete operation

Relates: #6259

Commit:
[TASK] add confirm dialogs to targeting delete operations

Relates: #6259

Commit:
[TASK] improve message of delete pimcore admin user

Relates: #6259

dpfahlbusch commented:
@brusch should be ready for review :)

Commit:
[TASK] add confirm dialog to static route delete operation

Relates: #6259

blankse commented:
Wouldn't it be better to just create one translation with 2 parameters?

Commit:
[TASK] refactor confirm delete message

Relates: #6259

blankse commented:
Can you use the translation function for the type parameter? So this can also be translated.

Commit:
[TASK] Use translations for type in delete confirmation

Relates: #6259

blankse reviewed (approved):
LGTM, thank you 👍🏼

Commit:
[TASK] Change icon of delete objectbrick to make it consistent with delete class and delete fieldcollection

Relates: #6259

dpfahlbusch commented:
Just added a minor change, because i was working on the menus:
changed the icon of delete fieldcollection (was in an earlier commit today) and delete objectbrick to make it consistent with delete class :)

Commit:
[BUGFIX] fix objectbrick menu

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/collectionsPanel.js:
```diff
@@ -311,15 +315,19 @@ pimcore.object.classificationstore.collectionsPanel = Class.create({
-                            success: function (response) {
-                                this.collectionsStore.reload();
-                            }.bind(this)});
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_collection'), data.data.text), function(btn) {
```
Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_collection'), data.data.text), function(btn) {

aryaantony92 replied:
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_collection'), data.data.name), function(btn) {
```

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/groupsPanel.js:
```diff
@@ -145,16 +145,20 @@ pimcore.object.classificationstore.groupsPanel = Class.create({
-                            success: function (response) {
-                                this.relationsStore.reload();
-                            }.bind(this)});
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_relation'), data.data.text), function(btn) {
```
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_relation'), data.data.keyName), function(btn) {
```

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/groupsPanel.js:
```diff
@@ -316,15 +320,19 @@ pimcore.object.classificationstore.groupsPanel = Class.create({
-                            success: function (response) {
-                                this.groupsStore.reload();
-                            }.bind(this)});
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_group'), data.data.text), function(btn) {
```
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_group'), data.data.name), function(btn) {
```

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/settings/staticroutes.js:
```diff
@@ -185,7 +185,11 @@ pimcore.settings.staticroutes = Class.create({
                             return;
                         }
-                        grid.getStore().removeAt(rowIndex);
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('staticroute'), data.data.text), function (btn) {
```
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('staticroute'), data.data.name), function (btn) {
```

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/collectionsPanel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/groupsPanel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/groupsPanel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/settings/staticroutes.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/propertiesPanel.js:
```diff
@@ -198,15 +198,19 @@ pimcore.object.classificationstore.propertiespanel = Class.create({
-                            success: function (response) {
-                                this.store.reload();
-                            }.bind(this)});
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_property'), data.data.text), function(btn) {
```
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_property'), data.data.name), function(btn) {
```

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/propertiesPanel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/collectionsPanel.js:
```diff
@@ -141,16 +141,20 @@ pimcore.object.classificationstore.collectionsPanel = Class.create({
-                            success: function (response) {
-                                this.relationsStore.reload();
-                            }.bind(this)});
+                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_collection_relation'), data.data.text), function(btn) {
```
```suggestion
                        Ext.Msg.confirm(t('delete'), sprintf(t('delete_message_advanced'), t('classificationstore_collection_relation'), data.data.groupName), function(btn) {
```

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/collectionsPanel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/settings/targeting/rules/panel.js:
```diff
@@ -192,15 +192,19 @@ pimcore.settings.targeting.rules.panel= Class.create({
+                    }.bind(this)
+                });
+            }
         });
```
```suggestion
       }.bind(this));
```

aryaantony92 commented on bundles/AdminBundle/Resources/public/js/pimcore/settings/targeting/targetGroups/panel.js:
```diff
@@ -161,17 +161,21 @@ pimcore.settings.targeting.targetGroups.panel= Class.create({
+                    }.bind(this)
+                });
+            }
         });
```
```suggestion
      }.bind(this));
```

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/settings/targeting/rules/panel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

dpfahlbusch committed:
Update bundles/AdminBundle/Resources/public/js/pimcore/settings/targeting/targetGroups/panel.js

Co-authored-by: aryaantony92 <97134765+aryaantony92@users.noreply.github.com>

aryaantony92 commented:
@dpfahlbusch Thank you
